src/main/resources/biz_de.properties

```diff
@@ -287,7 +287,9 @@ ImportMode.UPDATE_ONLY = Nur bestehende Datensätze aktualisieren
 IncidentController.totalIncidents = Anzahl Einträge
 InteractiveJobFactory.additionalMetrics = Weitere Werte
 InternationalAddressData.country = Land
+Isenguard.import = Importe
```

Comment:
The isenguard keys should be added in OX instead of biz, as the limits whith these names are only introduced in OX